src/main/java/com/blackducksoftware/integration/hub/alert/hub/controller/global/GlobalHubConfigController.java

```diff
@@ -57,26 +57,31 @@ public GlobalHubConfigController(final GlobalHubConfigActions configActions, fin
     }
 
     @Override
+    @PreAuthorize("hasRole('ADMIN')")
```

Comment:
I don't believe the "@PreAuthorize" is necessary for anything except the GET. This class has ConfigController in its hierarchy, which has the proper annotations. If this is not working, we probably have a bug in the all of the global controllers.